src/code/dialogs/stateDialog.js

```diff
@@ -16,7 +16,6 @@ const StateDialog = WDialog.extend({
 
   addHooks: function () {
     WDialog.prototype.addHooks.call(this);
-    this._setup();
```

Comment:
I've been meaning to remove all the _setups(), take care of the work in the constructor.